### ISSUE TITLE
Formatter fails on `@` inside PHP comment

### ISSUE THREAD

**Issue by zepfietje** [Stillat/blade-parser-typescript#79]
Sometimes the formatter fails on `@` inside a PHP comment.
Have had to work around this here: https://github.com/filamentphp/filament/pull/8670/commits/3fdd068e30195e4130a49f4d8144cac0c1db5c3c.

This is the error output from before that fix commit:

```
packages/support/resources/views/components/dropdown/header.blade.php
[error] packages/support/resources/views/components/dropdown/header.blade.php: SyntaxError: Unexpected end of input while parsing echo
[error] 
[error]   10| ])
[error]   11| 
[error]   12| <{{ $tag }}
[error]  >13|     {{
[error]   14|         $attributes
[error]   15|             ->class([
[error]   16|                 'fi-dropdown-header flex w-full gap-2 p-3 text-sm',
```

**Comment by JohnathonKoster** [comment]
Thanks for letting me know! Resolved in `1.6.14` 🙂

**Comment by zepfietje** [comment]
Perfect, thanks!
